pkg/controllers/resources/pods/scheduling/config.go

```diff
@@ -0,0 +1,34 @@
+package scheduling
+
+import (
+	"github.com/loft-sh/admin-apis/pkg/licenseapi"
+	"github.com/loft-sh/vcluster/pkg/pro"
+)
+
+type Config struct {
+	VirtualSchedulerEnabled bool
+	HybridSchedulingEnabled bool
+	HostSchedulers          []string
+}
+
+// NewConfig creates a new scheduling config with specified vCluster scheduling options. In the case of vcluster OSS
+// when Hybrid Scheduling is enabled, this func returns an error, because Hybrid Scheduling is a Pro-only feature.
+var NewConfig = func(virtualSchedulerEnabled, hybridSchedulingEnabled bool, _ []string) (Config, error) {
```

Comment:
Just out of interest. The reason for making this and the one below function literals instead of regular function declarations is to be able to overwrite them with vcluster-pro!?

Reply:
Yes, exactly :)